### ISSUE TITLE
RML Login [LO] fails for Rivendell users with no password set

### ISSUE THREAD

**Issue by vizubeat** [ElvishArtisan/rivendell#925]
I have a fresh installation of 4.1.2. No audio and nothing in the database other than a handful of users with different permissions to test issue #918. I can switch between users using `rdlogin` without issue, `rdairplay` reloads permissions as expected.

However, I also created 3 macro carts:

- `LO user!`
- `LO user1!`
- `LO user2!`

...so that I can add macro buttons to the Sound Panel to allow for easy user switching. When executing these carts, `rdairplay` appears to freeze or lose functionality, if not totally crash, and I see this in the logs:

```
==> /var/log/rivendell/operations <==
Nov 22 19:22:25 rd ripcd: received rml: "LO user1!" from 127.0.1.1

==> /var/log/syslog <==
Nov 22 19:22:26 rd mariadbd[723]: 2023-11-22 19:22:26 140 [Warning] Aborted connection 140 to db: 'Rivendell' user: 'rduser' host: 'localhost' (Got an error reading communication packets)
```

The only fix seems to be a restart of Rivendell services.

**UPDATE:** This was seen with users that had no password set in `rdadmin` - configuring a password for the users, and adding that password to the RML, e.g. `LO user password!` seems to work. So not urgent, as we should probably set passwords for users, but might be a nice thing to fix up in time.

**Comment by ElvishArtisan** [comment]
Confirmed here.

**Comment by ElvishArtisan** [comment]
Fixed in 982cdc2. Please test!
